deploy.sh

```diff
@@ -110,9 +110,11 @@ fi
 selectNodeVersion
 
 # 3. Install npm packages
+echo "beginning npm install"
```

Comment:
After we figured out our issue with deployment, we can add back the original --production